### PR TITLE
chrome-dev-tools cheatsheet added

### PR DESCRIPTION
**Pull request by Anuragmourya975**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/chrome-devtools-cheatsheet.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ created: 2022-10-20
 | :-------------: | :-: | :---------: |
 | `Ctrl + Delete(del)` | `Option(⌥) + Delete(del)` | Delete all characters in the last word, up to the cursor | 
 | `Ctrl + M` | `Ctrl + M` | Go to matching bracket |
+| `Ctrl+B` | `Cmd + B` | Add or remove a line-of-code breakpoint |
+| `Ctrl+D / Ctrl+U` | `Cmd+D / Cmd+U` | Select / de-select the next occurrence of whatever word the cursor is on. Each occurrence is highlighted simultaneously |
```
Comment (crescentpartha) [inline CheatSheets/chrome-devtools-cheatsheet.md:100]: Please, give the `reference link` to verify these commands.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, give the `reference link` to verify these commands.

File: CheatSheets/chrome-devtools-cheatsheet.md (hunk 1 above)
Comment (Anuragmourya975) [reply CheatSheets/chrome-devtools-cheatsheet.md:100]: https://developer.chrome.com/docs/devtools/shortcuts/#editor

Comment (Anuragmourya975) [reply CheatSheets/chrome-devtools-cheatsheet.md:102]: Sir, can you please review this PR

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**